### DISCUSSION TITLE
P1P2MQTT and charging

### DISCUSSION THREAD

**Discussion by sday** [Arnold-n/P1P2MQTT#149] [General]
Love this project. In rabbit hole exploration I am here looking for ways to charge a mini split without evacuating the system and charging by weight. JTAG ports... etc. Through the use of this project is there a way to communicate to the system that it should operate in such a way that you can get consistent accurate readings that would allow for charging during AC operation for single port minis? 

Everything I've read and watched so far say it's not possible. I find that hard to believe if you can instruct the motherboard to operate in max capacity mode for refrigerant level evaluation. 

Thanks for any input someone might share for this unexpected drive by posting. :) I have several 9k Mr. Cool units... not sure where my exploration will take me, but the goal is to master the maintenance and connect them through my own home grown automation system.

respectfully,
-Steve

**Comment by Arnold-n** [comment]
Thanks - but I do not think that it is possible - Mr. Cool is not supported by this project and we have not detected any refrigerant pressure data in P1/P2.

**Reply by sday** [reply]
Thanks for the input. I feel that my explanation wasn't sufficient. I was thinking this project might be able to command the system to max output. 

 I understand that on board systems do not detect refrigerant pressure data. In order to diagnose the level of refrigerant on a mini split with a single diagnostic port on the vapor side of the system, a technician can only test superheat to assess refrigerant level. Because modern efficient inverter driver mini splits use electronic expansion valves and adjust compressor and fan speed based on many variables no matter the manual setting, it is imperative for an accurate reading to involve maximum output so you can determine the superheat value. Although setting the unit to max cold and turbo mode conceptually sounds like it would do the trick, the units apparently still compute output levels based on perceived need. So for example if it's winter and the indoor temp is cooler and under pressure from outside to be cold, the unit may interpret that ease of cooling in a way that ramps the output down even with a max/turbo setting. So to circle back, I'm looking for a way to force a unit to deliver max output for a test period of 10-15 minutes to get a stable superheat reading. (rather than evacuating the system each time and re-weighing refrigerant when you charge it)

thanks again
-Steve

**Reply by sday** [reply]
In short, I need to command the compressor to max output, the EEV to max open and the indoor fan to max speed.

**Reply by Arnold-n** [reply]
Best you can do is set temperature target to max. But that can be done manually too.

**Reply by sday** [reply]
thanks for the input. Looks like I need to look deeper into JTAG control.
